blackbox/docs/admin/ssl.rst

```diff
@@ -45,9 +45,10 @@ the following steps:
 
 .. NOTE::
 
-  CrateDB monitors keystore and truststore files that are configured as values
-  of the node settings. If any of these files are updated CrateDB dynamically
-  reloads them. The file change polling frequency is 5 seconds.
+  CrateDB monitors SSL files such as keystore and truststore that are
+  configured as values of the node settings. If any of these files are updated
+  CrateDB dynamically reloads them. The polling frequency of the files is set
+  via the ``ssl.resource_poll_interval`` setting, which defaults to 5 seconds.
```

Comment:
For users using a `:ref:` here to the setting would probably be more convenient, then we could also avoid mentioning the 5 seconds here - which helps keeping the doc up2date